home/start_page.py

```diff
@@ -98,18 +97,28 @@ def read_config(self):
 
     def render(self):
         """Rendering all apps."""
-        self.output.clear_output()
-        apps = self.load_apps()
+        home = self._create_home_widget()
+        display(home)
+
+        from pathlib import Path
```

Comment:
Let's.move this import at the top of the module, not much point in inlining this imo. If you disagree, please at least move at the top of the function.

Reply:
I can't recall a good reason why it wasn't placed at the module's head. Will move up.